<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtMainWindow</class>
 <widget class="QMainWindow" name="qtMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>CDP || Bodegas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameDeBodegas">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:rgb(53, 53, 79);
}

QPushButton{
font:87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color:rgb(20, 200, 220);
border-radius:5px;
border:1px solid white;

}

QPushButton:hover{
background-color:black;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>852</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="qtBTNCreacionDeBodegas">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Creacion De Bodegas</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../Downloads/truck (1).svg</normaloff>../../Downloads/truck (1).svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qtBTNModificarInventarios">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Inventarios</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../Downloads/edit-3 (1).svg</normaloff>../../Downloads/edit-3 (1).svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qtBTNRegistroDeArticulos">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Registro De Articulos</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../Downloads/list (1).svg</normaloff>../../Downloads/list (1).svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qtBTNGestionDeBodegas">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Gestion De Bodegas</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../Downloads/folder (1).svg</normaloff>../../Downloads/folder (1).svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="qtMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu_Principal">
    <property name="title">
     <string>Menu Principal</string>
    </property>
    <addaction name="qtMNUSalir"/>
   </widget>
   <addaction name="menuMenu_Principal"/>
  </widget>
  <action name="actionCreacion_De_Bodegas">
   <property name="text">
    <string>Creacion De Bodegas</string>
   </property>
  </action>
  <action name="actionManejo_De_Bodegas">
   <property name="text">
    <string>Manejo De Bodegas</string>
   </property>
  </action>
  <action name="qtMNUSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCreacion_De_Bodegas_2">
   <property name="text">
    <string>Creacion De Bodegas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
